### PR TITLE
Finish second task

### PR DESCRIPTION
**Pull request by Ivanko5417**
[S3 Url ](http://my-rs-app-2.s3-website-us-east-1.amazonaws.com/)
[CloudFront Url](https://d2hi0j4dobntu1.cloudfront.net/)
[Automated configured CloudFront Link](https://djdhutrjn894.cloudfront.net/)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,10 @@ yarn-error.log*
 
 /.idea
 .vscode
+
+# package directories
+node_modules
```
Comment (ArthurKa) [inline .gitignore]: Дубликат строки 4.

File: .gitignore [hunk 2]
```diff
@@ -25,3 +25,10 @@ yarn-error.log*
...
+jspm_packages
+
+# Serverless directories
+.serverless
```
Comment (ArthurKa) [inline .gitignore]: Дубликат строки 13.

**Review by ArthurKa — COMMENTED** [review]
Битая ссылка на CloudFront. Забыл указать протокол.

File: serverless.yml [hunk 3]
```diff
@@ -0,0 +1,223 @@
+# Welcome to Serverless!
```
Comment (kaseopea) [inline serverless.yml]: Just small note! Would be nice to remove unused / commented out code

**Review by kaseopea — APPROVED** [review]

**Review by ArthurKa — APPROVED** [review]

**Pull request merged**